app/Http/Controllers/ReportsController.php

```diff
@@ -642,6 +642,9 @@ public function postCustom(Request $request)
             if (($request->filled('next_audit_start')) && ($request->filled('next_audit_end'))) {
                 $assets->whereBetween('assets.next_audit_date', [$request->input('next_audit_start'), $request->input('next_audit_end')]);
             }
+            if($request->filled('exclude_archived')){
+                $assets->where('archived', '=', 0);
```

Comment:
I think this wouldn't account for the assets that are not individually marked as archived, but instead belong to an archived status label. 

Maybe check out this query scope:
https://github.com/snipe/snipe-it/blob/3182f251e1b4a529069b8e6473f3f908cc267494/app/Models/Asset.php#L1062-L1067

Reply:
Accounts for status_id now as well 👍